src/org/traccar/notification/NotificationSms.java

```diff
@@ -20,33 +20,42 @@
 import org.traccar.model.Event;
 import org.traccar.model.Position;
 import org.traccar.model.User;
+import org.traccar.sms.SMSManager;
+import org.traccar.sms.SMSException;
 
-import com.cloudhopper.smpp.type.RecoverablePduException;
-import com.cloudhopper.smpp.type.SmppChannelException;
-import com.cloudhopper.smpp.type.SmppTimeoutException;
-import com.cloudhopper.smpp.type.UnrecoverablePduException;
+public final class NotificationSms extends Notificator {
 
-public final class NotificationSms {
+    private final SMSManager sms;
```

Comment:
I would recommend to rename variable to `manager` instead. SMS is just a message, so name is confusing.